leptos_macro/src/view.rs

```diff
@@ -1246,19 +1246,17 @@ fn attribute_to_tokens(
         let event_type = if is_custom {
             event_type
         } else if let Some(ev_name) = event_name_ident {
-            let span = ev_name.span();
-            quote_spanned! {
-                span => #ev_name
+            quote! {
```

Comment:
I have replaced `quote_spanned` with `quote`, since `quote`/ `quote_spanned` already preserve the `Span` of original tokens, and it's `_spanned` version only needed when new tokens are created.

https://docs.rs/proc-quote/latest/proc_quote/macro.quote_spanned.html#hygiene

Reply:
Excellent, thanks.